README.md

```diff
@@ -351,9 +351,28 @@ the private signing key, the verification key, and the wallet address on the
 preprod testnet. You can claim some **preprod** lovelaces using the
 [faucet](https://docs.cardano.org/cardano-testnet/tools/faucet/).
 
+Read the cbor hex for the private signing key using the following command:
+
+```shell
+cat bot.skey | jq -r '.cborHex'
+```
+
+This is the value assigned to `PAYMENT_SIGNING_KEY_CBOR_HEX` when running the SOR via Docker.
+
 It's **recommended** to create and setup a `collateral`. A UTxO with 5 ADAs will
 do the work. But as we mentioned the `collateral` config field is optional.
 
+Use the following command to find and select the UTXO to be used for collateral:
+
+```shell
+cardano-cli query utxo --address $(cat bot.preprod.addr) --testnet-magic 1
```

Comment:
Thanks a lot for your contribution @caike ! Maybe worth mentioning that this information can be conveniently obtained from explorer such as `cexplorer.io`.

Reply:
I would still prefer to add a note about mainnet or two commands: one for preprod and one for mainnet.

Reply:
Updated with these suggestions. However, the section of the README that shows how to create a wallet is using preprod:

```
cardano-cli address build \
    --payment-verification-key-file bot.vkey \
    --testnet-magic 1 \
    --out-file bot.preprod.addr
```

So maybe defaulting to `--mainnet` while using `bot.preprod.addr` as the address might be confusing. Thoughts ?